include/zephyr/pm/device.h

```diff
@@ -577,6 +577,16 @@ bool pm_device_wakeup_is_capable(const struct device *dev);
  */
 bool pm_device_on_power_domain(const struct device *dev);
 
+/**
+ * @brief Get the device's power domain.
+ *
+ * @param dev Device instance.
+ *
+ * @retval Device's power domain if on a power domain
```

Comment:
This is not the correct syntax for retval (see https://builds.zephyrproject.io/zephyr/pr/81793/docs/doxygen/html/group__subsys__pm__device.html#gab40a84454dd187de96fdb644756469d0)

```suggestion
 * @retval pd The device's power domain if on a power domain
```